### DISCUSSION TITLE
optimal way to read big data from two different dbs for comparison using sqlalchemy ?

### DISCUSSION THREAD

**Discussion by sivachaitanya** [sqlalchemy/sqlalchemy#9632] [Ideas]
thanks for an amazing library. Is there an efficient way to read same big data which has millions of rows in both mysql & postgres and then do data comparison between two outcomes without running into memory issues using sqlclachemy in my python script?

**Comment by CaselIT** [comment]
hi,

you can probably look into https://docs.sqlalchemy.org/en/20/core/connections.html#using-server-side-cursors-a-k-a-stream-results
